backend/qemu.pm

```diff
@@ -792,7 +792,10 @@ sub start_qemu {
         }
 
         if ($arch_supports_boot_order) {
-            if ($vars->{PXEBOOT}) {
+            if ($vars->{PXEBOOT} eq 'once') {
```

Comment:
Please see https://travis-ci.org/os-autoinst/os-autoinst/builds/574597957#L814 , you are now comparing against potentially undefined, better:

```perl
if (($vars->{PXEBOOT} // '') eq ...
```

Reply:
oh yeah, good spot.